lib/app/services/api_network.dart

```diff
@@ -0,0 +1,71 @@
+import 'package:flutter/widgets.dart';
+import 'package:get_it/get_it.dart';
+import 'package:meta/meta.dart';
+import 'package:http/http.dart' as http;
+import 'package:schulcloud/app/app.dart';
+
+import 'storage.dart';
+
+/// A service that offers networking POST and GET requests to the backend
```

Comment:
Only POST and GET requests is no longer correct—maybe just say "network requests"?